docs/Getting_Involved/Testing.rst

```diff
@@ -9,12 +9,28 @@ Executing our Tests
 *coala* has a big test suite. It is meant to work on every platform on
 every PC. If you just execute our tests you are doing us a favor.
 
-To run tests, you first need to install some dependencies. This can be
-done by executing:
+To run tests, you first need to install some dependencies.
+This can be done by following these steps:
+
+If you have not already, you can start by cloning a copy of the project
+to your local machine.
+
+Either fork the repo on Github https://github.com/coala-analyzer/coala,
+or clone it directly from coala:
```

Comment:
Well we're still cloning it from GitHub, not from coala. Also, they'd need to clone it nevertheless - you can't run tests on GitHub :P

So IMO I'd rather change the sentence like `You can clone the repository (or a fork of it) by running:` with `repository` having a link to our page.